src/js/navigation.js

```diff
@@ -62,12 +62,12 @@
   mobileNavWrapper.addEventListener('keydown', function(e) {
     if (e.key === 'Tab' || e.keyCode === 9) {
       if ( e.shiftKey ) /* shift + tab */ {
-        if (document.activeElement === firstFocusableEl) {
+        if (document.activeElement === firstFocusableEl && isDesktopNav() === false) {
```

Comment:
Just a minor code style thing... but you could simplify this to `(document.activeElement === firstFocusableEl && !isDesktopNav())`

Reply:
👍